node/pkg/guardiansigner/filesigner.go

```diff
@@ -0,0 +1,100 @@
+package guardiansigner
+
+import (
+	"crypto/ecdsa"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"google.golang.org/protobuf/proto"
+
+	nodev1 "github.com/certusone/wormhole/node/pkg/proto/node/v1"
+	"golang.org/x/crypto/openpgp/armor" // nolint
```

Comment:
What's the lint issue here?

Reply:
This is because "golang.org/x/crypto/openpgp/armor" is deprecated. We do the same thing in `armoredKey.go:LoadGuardianKey()`.